src/TableColumn/Status.php

```diff
@@ -4,91 +4,13 @@
 
 namespace atk4\ui\TableColumn;
 
-use atk4\data\Model;
+if (!class_exists(\SebastianBergmann\CodeCoverage\CodeCoverage::class, false)) {
+    'trigger_error'('Use atk4\ui\Table\Column\Status instead', E_USER_DEPRECATED);
+}
 
 /**
- * Implements Column helper for grid.
+ * @deprecated will be removed jun-2021
```

Comment:
this needs to be updated, PR rebased and at least from my side LGTM

Reply:
updated